telegram-mini-app-game/README.md

```diff
@@ -0,0 +1,105 @@
+# 🏀🎮 Telegram Mini App Game Bounty
+
+## 🌟 Overview
+
+We're thrilled to announce a bounty of up to **$ ** for creating an engaging Telegram Mini App Game using Spheron's decentralized GPU network and integrating Web3 components! 🚀
+
+This game should:
+
+- 🎮 Be playable within Telegram
+- 🖥️ Utilize decentralized GPU resources for enhanced functionality
+- 🌐 Incorporate Web3 elements and be deployable on any blockchain
+
+Our goal is to develop an innovative and captivating game that leverages Spheron's decentralized GPU network and Web3 technologies, providing a seamless and immersive gaming experience. 🎲✨
+
+## 🎯 Objective
+
+Develop a Telegram Mini App Game that can:
+
+1. 🎮 Provide an engaging and enjoyable gaming experience within Telegram
+2. 🖥️ Utilize decentralized GPU resources for game processing and functionality
+3. 🌐 Integrate smoothly with the Telegram Mini App framework
+4. 🌍 Incorporate Web3 components and be deployable on any blockchain
+
+## 📋 Requirements
+
+### 1. Gameplay
+
+- 🎯 Develop a captivating and interactive game
+- 🕹️ Ensure smooth gameplay and user interaction
+
+### 2. GPU Utilization
+
+- 🖥️ Leverage Spheron's decentralized GPU network for game functionality
+- ⚡ Optimize game performance using GPU resources
+
+### 3. Telegram Integration
+
+- 📱 Build the game as a Telegram Mini App
+- 🌐 Ensure seamless integration and functionality within the Telegram environment
+
+### 4. Web3 Integration
+
+- 🌍 Incorporate Web3 elements (e.g., NFTs, token rewards, smart contracts)
+- ⛓️ Enable deployment on any blockchain of your choice
+
+### 5. User Interface
+
+- 💻 Develop an intuitive and user-friendly UI as Telegram Mini App
+- 🖌️ Provide engaging graphics and visual elements
+
+### 6. Technology Stack
+
+- 🔧 Backend: Any preferred language (e.g., Python, Go, Node.js)
+- 🖥️ Frontend: Web application or mobile app
+- 🛠️ Utilize Spheron's decentralized GPU network, Telegram Mini App, and Web3 technologies
+
+### 7. Documentation
+
+- 📚 Comprehensive setup, configuration, and usage instructions
+- 📝 Guidelines for extending the app to support additional features and improvements
+
+### 8. Testing
+
+- 🧪 Robust testing for various edge cases
+- ✅ Include unit tests, integration tests, and end-to-end tests
+
+## 💰 Bounty Details
+
+Participants can earn up to **$ ** based on the implemented features.
+
+## 📤 Submission Guidelines
+
+1. Create an issue in the [bounty repository](https://github.com/spheronfdn/spheron-bounties) including:
```

Comment:
Hey @HAPPYS1NGH the issue submission link attached is wrong. It must be https://github.com/spheronFdn/sos-ai-bounty

Please correct it

Reply:
Fixed that